apps/web/src/app/[orgShortcode]/convo/_components/new-convo-sheet.tsx

```diff
@@ -43,12 +47,30 @@ export function NewConvoSheet() {
           </DrawerDescription>
         </DrawerHeader>
 
-        <Button
-          variant={'ghost'}
-          onClick={() => setOpen(false)}
-          className="fixed right-1 top-1">
-          <X className="h-4 w-4" />
-        </Button>
+        <div className="absolute right-1 top-1 flex gap-0 p-1">
+          <Button
+            size={'icon'}
+            variant={'ghost'}
+            onClick={() => {
+              router.push(`/${orgShortcode}/convo/new`);
```

Comment:
need to make sure to save the draft before maximizing

Reply:
@BlankParticle

Reply:
the draft is saved every 500ms, if you stop typing and click on the fullscreen button, the draft would be saved by then anyways